Source/Blazorise/Components/Droppable/DropContainer.razor.cs

```diff
@@ -75,7 +75,7 @@ public async Task CommitTransaction( string dropZoneName, bool reorderIsAllowed
         {
             index = GetTransactionIndex() + 1;
 
-            if ( transaction.SourceZoneName == transaction.CurrentZoneName && IsItemMovedDownwards )
+            if ( transaction.SourceZoneName == transaction.CurrentZoneName && IsItemMovedDownwardsOrSame )
             {
                 index -= 1;
```

Comment:
There isn't much to review. I'm not sure what this code does but just to make sure...
On line 76 you have increased the index by 1 and on line 80 you decrease it again by 1. Will it not pretty much stay the same as GetTransactionIndex? Or is that expected? Also if it stays the same. Maybe the logic should change to be more clear where it really only increases by 1 given a certain condition. 
And does it need to be set to some internal state?

Reply:
`+1` is because once the item is dropped, it is assumed it will be placed after the currently focused item.

Reply:
but if you do 
**index = GetTransactionIndex() + 1;**
and then
**index -= 1;**
It will just go back to the original transaction index, and seems harder to reason? Which means, maybe the logic could be simplified to a single if, or something?

i.e: 
`index = GetTransactionIndex();`
`if (logic)`
`  index += 1; `

Reply:
Yeah, that seems better.